src/helpers/issue.ts

```diff
@@ -67,6 +67,24 @@ export const listIssuesForRepo = async (state: "open" | "closed" | "all" = "open
   }
 };
 
+export const listAllIssuesForRepo = async (state: "open" | "closed" | "all" = "open") => {
+  const issuesArr = [];
+  let fetchDone = false;
+  const perPage = 30;
```

Comment:
30 seems unusual is this confirmed?

I would maybe expect 25 due to the notifications view paginating every 25.

Reply:
sure, fixed.

Reply:
rate limitting added

Reply:
It doesn't matter actually, we can set any random number for that.